src/Features/Core/Portable/Completion/Providers/AbstractOverrideCompletionProvider.ItemGetter.cs

```diff
@@ -112,67 +113,20 @@ private partial class ItemGetter
             }
 
             private bool TryDetermineOverridableMembers(
-                SemanticModel semanticModel, SyntaxToken startToken, Accessibility seenAccessibility, out ISet<ISymbol> overridableMembers)
+                SemanticModel semanticModel, SyntaxToken startToken,
+                Accessibility seenAccessibility, out ImmutableArray<ISymbol> overridableMembers)
             {
-                var result = new HashSet<ISymbol>();
-
                 var containingType = semanticModel.GetEnclosingSymbol<INamedTypeSymbol>(startToken.SpanStart, _cancellationToken);
-                if (containingType != null && !containingType.IsScriptClass && !containingType.IsImplicitClass)
-                {
-                    if (containingType.TypeKind == TypeKind.Class || containingType.TypeKind == TypeKind.Struct)
-                    {
-                        var baseTypes = containingType.GetBaseTypes().Reverse();
-                        foreach (var type in baseTypes)
-                        {
-                            _cancellationToken.ThrowIfCancellationRequested();
-
-                            // Prefer overrides in derived classes
-                            RemoveOverriddenMembers(result, type);
-
-                            // Retain overridable methods
-                            AddOverridableMembers(result, containingType, type);
-                        }
-
-                        // Don't suggest already overridden members
-                        RemoveOverriddenMembers(result, containingType);
-                    }
-                }
+                var result = containingType.GetOverridableMembers(_cancellationToken);
 
                 // Filter based on accessibility
                 if (seenAccessibility != Accessibility.NotApplicable)
```

Comment:
So, 'override completion' allows for the user to type something like "public override <space>".  In that case, we filter the overrides to the set of ones that match the accessibility written.  There's no need for that in the 'generate overrides...' case, so i didn't put any accessibility check there.